src/ComputationContainer/Client/ComputationToDbGate/ValueTable.cpp

```diff
@@ -74,49 +78,152 @@ std::vector<std::pair<int, int>> intersectionSortedValueIndex(
     const std::vector<T> &sorted_v1, const std::vector<T> &sorted_v2
 )
 {
-    // v1, v2がソートされている必要あり
-    std::vector<std::pair<int, int>> it_list;
-    it_list.reserve(sorted_v1.size());
-
-    size_t i1 = 0, i2 = 0;
-    std::uint32_t iterated = 0;
+    // v2がソートされている必要あり
```

Comment:
I thought `v1` should still be sorted. Is this wrong?

Reply:
You are right.
I fixed it in 122eb39d4496a4963c28053a11b16d25a6676f0d!

Reply:
thanks!